packages/api/test/cors.spec.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-env mocha, browser */
+import assert from 'assert'
+
+const endpoint = 'http://testing.filecoin.storage'
```

Comment:
There is a magic endpoint string in the baked into nft.storage router, but that isn't the case here. This test fails if you have no network connection. I think it's trying to hit ipfs.io to fetch the CAR file and fails becuase `res.ok` is not ok.